packages/util-crypto/src/index.ts

```diff
@@ -17,8 +17,10 @@ export * from './keccak';
 export * from './key';
 export * from './mnemonic';
 export * from './nacl';
+export * from './pbkdf2';
```

Comment:
Do you still want to keep it, even despite PBKDF2 is not being used anymore?

Reply:
Yes, since it is actually used elsewhere it should have been exposed already and it wasn't.